v19.2/cockroachcloud-sign-up-for-a-cluster.md

```diff
@@ -13,29 +13,30 @@ The CockroachCloud offering provides a running CockroachDB cluster suitable to y
 - Upgrades to the latest stable release of CockroachDB
 - Monitoring to provide SLA-level support
 
-We are offering initial access to select organizations through our beta program. [Sign up here](https://www.cockroachlabs.com/product/cockroachcloud/) for early access to CockroachCloud.
+We are offering initial access to select organizations through our beta program. [Sign up here](https://www.cockroachlabs.com/product/cockroachcloud/) for early access to CockroachCloud. If you are selected for the beta program, we will notify you via email.
 
-## Set your password
+## Create your CockroachCloud account
 
-Once your CockroachCloud cluster is available, you'll receive an email prompting you to set a password and sign in:
+Once you are accepted for the beta program, you'll receive an email prompting you to create your CockroachCloud account:
 
-1. In the confirmation email, click **Set password**.
-
-    The **Set Password** page displays.
-
-    <img src="{{ 'images/v19.2/cockroachcloud/sign-up.png' | relative_url }}" alt="Sign up" style="border:1px solid #eee;max-width:100%" />
-
-2. Enter your name.
-3. Enter and confirm your password.
-4. Click **Submit**.
-
-    The [**Clusters**](cockroachcloud-clusters-page.html) page displays.
+1. Click on the **Create Account** link in the email to access the **Create your account** page.
+2. On the **Create your account**, enter your first and last name, email address, and password.
```

Comment:
This might need to say "**Create your account page**..." or something. _"On the Create your account,"_ sounds incomplete.